metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1168,6 +1168,7 @@ class MetalsLspService(
     compilers.didClose(path)
     trees.didClose(path)
     diagnostics.onClose(path)
+    interactiveSemanticdbs.onClose(path)
```

Comment:
There is also `def onDelete`, shouldn't this be done there too?

Reply:
I think this should be fine. If it's deleted and was open we will first get close notification. If a file is deleted that is not open it will not end up in InteractiveSemanticdb in the first place.